test/test_file.py

```diff
@@ -24,20 +25,55 @@ def write_bzImage(path: Path,
         f.write(version_line)
 
 
+def write_raw(path: Path,
+              version_line: bytes
+              ) -> None:
+    """Write a raw kernel binary at `path`, with `version_line`"""
+    with open(path, 'wb') as f:
+        f.write(0x210 * b'\0')
+        f.write(version_line)
+
+
+def write_compress(path: Path,
+                   version_line: bytes
+                   ) -> None:
+    """Write a gzip compressed raw kernel binary at `path`,"""
+    """with `version_line`"""
+    with gzip.open(path, 'wb') as f:
+        # gzip would compress a string of 0s below 0x200,
+        # so we fill in some incompressible gibberish
+        f.write(os.urandom(0xffff))
```

Comment:
This doesn't really guarantee it will be 'incompressible'. It would be better to use some predictable algorithm, like playing with `range()`.